src/prettify.js

```diff
@@ -141,7 +141,8 @@ var PR;
  * UI events.
  * If set to {@code false}, {@code prettyPrint()} is synchronous.
  */
-window['PR_SHOULD_USE_CONTINUATION'] = true;
+var PR_SHOULD_USE_CONTINUATION = true;
+if (typeof window !== 'undefined') window.PR_SHOULD_USE_CONTINUATION = PR_SHOULD_USE_CONTINUATION;
```

Comment:
Sure. I will check it out. btw. wouldn't it be more logical to have the sources in the `src` folder and the destination in a `dist` one?